app/src/main/java/org/glucosio/android/tools/GlucosioConverter.java

```diff
@@ -23,7 +23,7 @@
 import java.math.BigDecimal;
 import java.math.RoundingMode;
 
-public class GlucosioConverter {
+public abstract class GlucosioConverter {
```

Comment:
Final with private constructor will work better. Right now you can still extend it and instantiate child class.